### PR TITLE
Martin bardarov

### PR DESCRIPTION
**Pull request by martinbardarov-exe**

### CODE REVIEW

File: Loops_MartinBardarov/IsPalindrome.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class IsPalindrome {
+
+    public boolean isPalindrome(String word) {
```
Comment (Turnikman88) [inline Loops_MartinBardarov/IsPalindrome.java:3]: word needs to be Integer, then converted to String

File: Loops_MartinBardarov/IsPalindrome.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        boolean reversed = false;
+        StringBuilder input1 = new StringBuilder();
+        input1.append(word);
+        input1.reverse();
```
Comment (Turnikman88) [inline Loops_MartinBardarov/IsPalindrome.java:7]: Can you solve the problem without using this method?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
-# Blankfactor_Interns_Basics
```
Comment (Turnikman88) [inline README.md:1]: remove the changes

**Review by Turnikman88 — CHANGES_REQUESTED** [review]

**Comment by martinbardarov-exe** [comment]
Done

**Pull request closed without merging**